data/2mass_j12560183-1257276.json

```diff
@@ -69,22 +179,72 @@
             "reference": "Dupu20"
         }
     ],
+    "Spectra": [
+        {
+            "spectrum": "https://bdnyc.s3.amazonaws.com/nir_spectra/vhs1256b_nir_SOFI.fits",
+            "original_spectrum": "https://bdnyc.s3.amazonaws.com/vhs1256b_optnir_werrors.txt",
+            "local_spectrum": null,
+            "regime": "nir",
+            "telescope": "NTT",
+            "instrument": "SofI",
+            "mode": null,
+            "observation_date": "2014-03-12T00:00:00",
+            "wavelength_units": "Angstrom",
+            "flux_units": "Angstrom-1 cm-2 W",
+            "wavelength_order": null,
+            "comments": null,
+            "reference": "Gauz15",
+            "other_references": null
+        },
+        {
+            "spectrum": "https://bdnyc.s3.amazonaws.com/nir_spectra/vhs1256b_spectra_Figure8_Miles2018.fits",
+            "original_spectrum": "https://bdnyc.s3.amazonaws.com/nir_spectra/vhs1256b_spectra_Figure8_Miles2018.txt",
+            "local_spectrum": null,
+            "regime": "nir",
+            "telescope": "Keck II",
+            "instrument": "NIRSPEC",
+            "mode": null,
+            "observation_date": "2016-06-19T00:00:00",
+            "wavelength_units": "um",
+            "flux_units": "cm-2 um-1 W",
+            "wavelength_order": null,
+            "comments": null,
+            "reference": "Mile18",
+            "other_references": null
+        },
+        {
+            "spectrum": "https://bdnyc.s3.amazonaws.com/optical_spectra/vhs1256b_opt_Osiris.fits",
+            "original_spectrum": "https://bdnyc.s3.amazonaws.com/vhs1256b_optnir_werrors.txt",
+            "local_spectrum": null,
+            "regime": "optical",
+            "telescope": "GTC",
+            "instrument": "OSIRIS",
+            "mode": null,
+            "observation_date": "2014-06-03T00:00:00",
+            "wavelength_units": "Angstrom",
+            "flux_units": "Angstrom-1 cm-2 W",
+            "wavelength_order": null,
+            "comments": null,
+            "reference": "Gauz15",
+            "other_references": null
+        }
+    ],
     "SpectralTypes": [
         {
-            "spectral_type_string": "L7:: VL-G",
+            "spectral_type_string": "L7 VL-G",
```

Comment:
Why this change?

Reply:
No one uses the double colons to indicate uncertainty except me.